platform/linux-generic/odp_crypto_ipsecmb.c

```diff
@@ -34,6 +34,7 @@
 #define IPSEC_MB_CRYPTO_MAX_AUTH_KEY_LENGTH        32
 #define IPSEC_MB_CRYPTO_MAX_DATA_LENGTH            65536
 #define ZUC_DIGEST_LENGTH 4
+#define SNOW3G_DIGEST_LENGTH ZUC_DIGEST_LENGTH
```

Comment:
This should be defined as just 4 as snow3g does not really have anything to do with zuc.

Reply:
Thanks for review, updated.